FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,16 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else {
+				return
+			}
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, from: response))
+			case .failure:
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
The `Error` is declared under`RemoteFeedLoader`, so `RemoteFeedLoader` can be omitted in this expression.

```suggestion
				completion(.failure(Error.connectivity))
```